### DISCUSSION TITLE
使用privategpt后输出log可以关闭吗

### DISCUSSION THREAD

**Discussion by BondChang** [ymcui/Chinese-LLaMA-Alpaca#589] [Q&A]
1.llama本身的log可以关闭吗？
![image](https://github.com/ymcui/Chinese-LLaMA-Alpaca/assets/32505512/095b94c2-b242-4906-a96e-f00e92f291b8)

2.当我使用教程中的优化策略时，会有额外的答案输出，可以关掉吗

![image](https://github.com/ymcui/Chinese-LLaMA-Alpaca/assets/32505512/9413b0d5-540a-4479-babc-ac1c421a2269)

![image](https://github.com/ymcui/Chinese-LLaMA-Alpaca/assets/32505512/d63bd7f0-dd74-49ba-b24f-af23f871152c)

**Comment by ymcui** [comment]
据我所知没有相应开关，要么就是自己改代码。
建议你去对应第三方库找找解决方案。
